eng/common/instructions/azsdk-tools/sdk-details-in-release-plan.instructions.md

```diff
@@ -5,13 +5,16 @@ description: 'Identify languages configured in the TypeSpec project and add it t
 **Goal**: Identify languages configured in the TypeSpec project and generate the json object with language and package name.
 1. Identify the language emitter configuration in the `tspconfig.yaml` file in the TypeSpec project root.
 2. Identify the package name or namespace for each language emitter.
+   - For Java and Python, use `package-dir` to get the package name.
```

Comment:
package-dir is in the process of being phased out and instead will be using emitter-output-dir (https://github.com/Azure/azure-sdk-tools/issues/10995)